### DISCUSSION TITLE
Welcome to cockpit-machines Discussions!

### DISCUSSION THREAD

**Discussion by KKoukiou** [cockpit-project/cockpit-machines#76] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by anand3132** [comment]
how to uninstall cockpit-machines..?

**Reply by martinpitt** [reply]
How did you install it in the first place? We usually recommend to install your Linux distribution's package, just uninstall that again?
